.drone.yml

```
@@ -66,6 +66,7 @@ steps:
 - name: test-backend
   image: grafana/build-container:1.2.28
   commands:
+  - yarn grunt no-focus-convey-tests
```

What does `yarn grunt no-focus-convey-tests` do? Why should it run for backend tests?

---

I think we can replace it with this:
```suggestion
  - [[ $(grep FocusConvey -R pkg | wc -l) -eq "0" ]] || exit 1
```